packages/react-router/modules/__tests__/integration-test.js

```diff
@@ -38,6 +38,79 @@ describe('Integration Tests', () => {
     expect(div.innerHTML).toNotContain(TEXT)
   })
 
+  it('renders an exact route with trailing slash', () => {
```

Comment:
Should there also be a test of the inverse? `<Route path='/somepath/'>` when `location.pathname = '/somepath'`?